explainer.md

```diff
@@ -331,24 +331,53 @@ customElements.define("custom-tablist", CustomTabList);
+| `focus`               | _all focusable elements_                                          |                       | `focus`                            |
+| `select`              | Elements whose computed role supports `aria-selected`             |                       | `click`                            |
+| `dismiss` or `escape` | _all elements_                                                    |                       | `Escape` KeyboardEvent             |
+| `contextMenu`         | _all elements_                                                    |                       | `contextmenu`                      |
+| `increment`           | Elements w/ computed role `progressbar`, `scrollbar`, or `slider` | vertical              | `Up` KeyboardEvent                 |
+|                       | ""                                                                | horizontal LTR        | `Right` KeyboardEvent              |
+|                       | ""                                                                | horizontal RTL        | `Left` KeyboardEvent               |
+|                       | Elements w/ computed role `spinbutton`                            | orientation n/a       | `Up` KeyboardEvent                 |
+| `decrement`           | Elements w/ computed role `progressbar`, `scrollbar`, or `slider` | vertical              | `Down` KeyboardEvent               |
+|                       | ""                                                                | horizontal LTR        | `Left` KeyboardEvent               |
+|                       | ""                                                                | horizontal RTL        | `Right` KeyboardEvent              |
+|                       | Elements w/ computed role `spinbutton`                            | orientation n/a       | `Down` KeyboardEvent               |
+| `scrollByPage`        | TBD (possibly custom scroll views)                                |                       | TBD (possibly `PageUp`/`PageDown`) |
+| `scrollIntoView`      | TBD                                                               |                       | No equivalent DOM event            |
+| `setValue`            | n/a                                                               |                       | No equivalent DOM event            |
+
+Notes on the previous table:
+- KeyboardEvent sequences include keyup/keydown/keypress.
+- Control orientation is determined by the computed value of `aria-orientation` which 
+  defaults to `horizontal` for `progressbar` and `slider`, and defaults to `vertical` for 
+  `scrollbar`.
+- Natural language direction is determined by the computed value of `dir` which defaults 
+  to `ltr`, but can be set to `rtl` for languages like Arabic and Hebrew.
+- The event target for KeyboardEvent sequences is the currently focused element.
```

Comment:
Actually, this is interesting. What if the AT sends an event on an element which is not focused? Should it be dropped?

Reply:
I assume sending a keyboard event on something is not the document.activeElement would be a non-starter, since that would clearly indicate a difference in the user environment.

Reply:
Maybe a better answer to your question is: 
- If it's something like a leaf node (slider) that is not focused, then yes, I think no keyboard event should be sent.
- If it's something like a dismiss event on a container, it can always be sent, regardless if the AT is on something that is focusable

Reply:
In the second case, the event should be re-targeted to document.activeElement?

Reply:
Yes I think so. If a goal is to make it as close to keyboard operation as possible, that's where the key events originate, so AT-triggered key events should also originate from document.activeElement, right?

Reply:
I think we should be specific about how that retargeting happens, so that we can be sure we're doing the same thing.